components/application/route.js

```diff
@@ -30,7 +30,8 @@ Route.parse = routeString => {
   const s = routeString.charAt() === '/' ? routeString.slice(1) : routeString
 
   // Skip '/data-providers/' part
-  const [, dataProvider, activity] = s.split('/')
+  const [, dataProvider, ...activityPartials] = s.split('/')
+  const activity = activityPartials.join('/')
```

Comment:
@Joozty by the way, `.split('/', 3)` does not work because of the following (from the [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)):

> If provided, splits the string at each occurrence of the specified `separator`, but stops when `limit` entries have been placed in the array. **Any leftover text is not included in the array at all.**